### DISCUSSION TITLE
where can i find the ml_Synth library without it i wont be able to compile it seems, but i do not know where to find it!?!

### DISCUSSION THREAD

**Discussion by fabiendostie** [marcel-licence/esp32_basic_synth#79] [Q&A]
Hello all, it's my first post here I really need some help.

I have been struggling with this for a while now 3 weeks actually, so i'm asking for help I feel really dumb for asking but I just cant figure this out.
I have watched all the videos, read all the readme's i am a little desperate I have to be honest, I am not that experienced, but at the same time i've built a few midi controllers using arduino's and esp's, my whole garden irrigation is fully automated using a bunch of esp32's and 8266's  so i can code a little, enough to get me by usually, but For the life of me I just cant figure out what I'm missing with this project!!! I'm stuck and I am begging for help.

Where are those files? where do I get access to them to modify them for my setup?
where can i find the ml_Synth library without it i wont be able to compile it seems, but i do not know where to find it!?!
Where are: ml_boards.h, ml_arp.h, ml_reverb.h, ml_midi_ctrl.h, ml_delay.h, ml_scope.h?

Thanks in advance, a sad thinkerer and musician.

**Comment by marcel-licence** [comment]
Is this the same issue like described here: https://github.com/marcel-licence/esp32_fm_synth/issues/9 ?

**Comment by nmaron** [comment]
If you click on the link in the source code in arduino ide it will take you right to the GitHub repos. Click on Code and download the zip file. The using the library manager in the ide just add the zip file and you’ll be all set.

**Reply by marcel-licence** [reply]
I am not sure if that could also help: https://www.youtube.com/watch?v=c7TL8jcrnzs
